### ISSUE TITLE
Differences in metrics rmse / rmse_default and what is me_default_sigma in the Report json output

### ISSUE THREAD

**Issue by GwenVCX** [evidentlyai/evidently#907]
I am using your library to work out when to send an alert. My colleagues, wants me to send an alert when the RMSE value is high or the z-score or the sigma gets out of the 2 standard deviations. I want to use the Report json output for these values, however, the metrics are unclear. Could you please clarify the following?

1) What is the difference between rmse and rmse_default and which one should I use?
2) What is me_default_sigma? Can I use that if I am using rmse?

Thank you.

**Comment by emeli-dral** [comment]
Hi @GwenVCX ,
Thank you for your interest!

1. What is the difference between rmse and rmse_default and which one should I use?
The rmse is a root mean squared error, which is calculated on top passed target and prediction data. This is a standard metric. If you something like ```sklearn.mean_squared_error(target, prediction, squared=False)``` for your data, you will get this rmse value.
However, together with this standard rmse we also calculate a root mean squared error for a "dummy model", so that one can compare the current rmse with a baseline. In our case dummy model is the model, which always predicts a mean target value for each object. So rmse_default is a error which you will get if you calculate an average target value on you dataset, and then use it as a model prediction for all objects.

2. What is me_default_sigma? Can I use that if I am using rmse?
me_default_sigma is a standard deviation around prediction error. This is somewhat like ```(target- prediction).std()``` . I would not go for me_default_sigma instead of rmse because me_default_sigma does not give as any information about mean error itself. It gives us only information about how much errors are deviated from the mean error, however this mean error can possibly be high or low.

I hope it has helped to clarify this things!
